CONTRIBUTORS.txt

```diff
@@ -134,3 +134,5 @@ Contributors
 - Antti Haapala, 2016/01/17
 - Bert JW Regeer, 2016/02/16
 - Steve Piercy, 2016/02/26
+- Josip Delic, 2016/04/01
+- Marc Rijken, 2016/04/01  
```

Comment:
Where did Marc Ripken add his name to CONTRIBUTORS.txt, since it is a semi-legal document we can't just have someone add their name CONTRIBUTORS.txt!